<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistributeDialog</class>
 <widget class="QDialog" name="DistributeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSurface">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>16</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ ОПВНО</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonSurface">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonAir">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAir">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>56</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ ПвО</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSubSurface">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>96</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ ПдО</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonSubSurface">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCMS">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>136</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ ПБУ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonCMS">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonAGS2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>94</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAGS1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ АУМ1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonAGM">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>14</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAGM">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ АУС</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButtonAGS1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>54</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Завантажити</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAGS2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АРМ АУМ2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkConnectionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Перевірити з’єднання</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistributeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DistributeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
